src/de/jost_net/JVerein/Variable/SpendenbescheinigungMap.java

```diff
@@ -35,7 +35,7 @@
 import de.jost_net.JVerein.util.JVDateFormatTTMMJJJJ;
 import de.jost_net.JVerein.util.StringTool;
 import de.willuhn.logging.Logger;
-import jonelo.NumericalChameleon.SpokenNumbers.GermanNumber;
+import net.numericalchameleon.util.spokennumbers.GermanNumber;
```

Comment:
Das ist wegen einer lib aus einem maven repository notwendig, richtig?

Reply:
NumericalChameleon gibt es in der Version wie sie im lib folder lag nicht. Ich habe die aktuellste Version von sourceforge gezogen und diese genommen. Der Autor hat aber inzwischen auf Standard Java Konventionen umgestellt und den packagenamen geändert.